jep/3/README.adoc

```diff
@@ -0,0 +1,182 @@
+= JEP-3: :bulb: Adding the "Released As" field and "Ready for Release" status to Jenkins JIRA :bulb:
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 3
+
+| Title
+| :bulb: Title :bulb:
+
+| Sponsor
+| link:https://github.com/oleg-nenashev[@oleg-nenashev]
+
+| Status
+// Uncomment the appropriate line.
+| Draft :information_source:
+//| Draft :speech_balloon:
+//| Deferred :hourglass:
+//| Accepted :ok_hand:
+//| Rejected :no_entry:
+//| Withdrawn :hand:
+//| Final :lock:
+//| Replaced :dagger:
+//| Active :smile:
+
+| Type
+| :bulb: Standards, Informational, or Process :bulb:
+
+| Created
+| Jan 29, 2018
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+| JIRA
+| https://issues.jenkins-ci.org/browse/INFRA-1301[INFRA-1301]
+//
+//
+// Uncomment if there will be a BDFL delegate for this JEP.
+//| BDFL-Delegate
+//| :bulb: Link to github user page :bulb:
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+| Discussions-To
+| link:https://groups.google.com/forum/#!topic/jenkinsci-dev/wzc4VLplHvs[jenkinsci-dev thread]
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+
+== Motivation
```

Comment:
Please see https://github.com/jenkinsci/jep/blob/master/jep/1/README.adoc#required-sections
This section should be the "Abstract".  

Also, unless there's a strong reason to deviate you should retain the suggested order, with "Motivation" after the "Specification".